src/test/java/com/skalashynski/spring/springboot/service/StudentServiceTest.java

```diff
@@ -0,0 +1,231 @@
+package com.skalashynski.spring.springboot.service;
+
+import com.skalashynski.spring.springboot.entity.Student;
+import com.skalashynski.spring.springboot.exception.ApiException;
+import com.skalashynski.spring.springboot.repository.StudentRepository;
+import com.skalashynski.spring.springboot.service.impl.StudentServiceImpl;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+import static org.mockito.BDDMockito.*;
```

Comment:
Don't import all classes/static methods. Configure IDEA to use imports only for necessary objects.

Reply:
done